lib/griddler/sendgrid/adapter.rb

```diff
@@ -26,6 +27,33 @@ def recipients(key)
         ( params[key] || '' ).split(',')
       end
 
+      def get_bcc
+        return [] unless params[:envelope].present?
+        envelope = json_to_hash(params[:envelope])
+        bcc = if envelope.present?
```

Comment:
@gabebw I prefer this way:

```ruby
bcc = envelope["to"] || []
```

IMHO, looks more readable than the `.fetch` method.